tools/docker/build_wheel.Dockerfile

```diff
@@ -1,7 +1,7 @@
 #syntax=docker/dockerfile:1.1.5-experimental
-ARG TF_VERSION
 ARG PY_VERSION
-FROM tfaddons/tensorflow:2.1.0-custom-op-gpu-ubuntu16-minimal as base_install
+ARG IMAGE_TYPE
+FROM tensorflow/tensorflow:2.1.0-custom-op$IMAGE_TYPE-ubuntu16 as base_install
```

Comment:
You can change the namespace to `tfaddons` when you will have a new squashed image.